README.md

```diff
@@ -1,15 +1,127 @@
+
+To learn more about the project, check out this [blog post](https://www.whitehouse.gov/blog/2016/08/08/peoples-code).
+
+Code.gov is an open source project, so we invite your contributions, be it in the form of code, design, or ideas.
+
+## Requirements
+
+The development of code.gov is guided by the requirements set forth in [Section 7.2 (Code Inventories and Discovery)](https://sourcecode.cio.gov/Implementation/#code-inventories-and-discovery), [Section 7.3 (Code.gov)](https://sourcecode.cio.gov/Implementation/#codegov), and  [Section 7.6 (Agency Policy)](https://sourcecode.cio.gov/Implementation/#agency-policy) of the Federal Source Code Policy. Namely:
+
+> * "Within 90 days of the publication date of this policy, the Administration will launch https://www.code.gov, an online collection of tools, best practices, and schemas to help agencies implement this policy.";
+>
+> * "Within 90 days of the publication date of this policy, each agency’s CIO—in consultation with the agency’s CAO—shall develop an agency-wide policy that addresses the requirements of this [document.]";  and
+>
+> * "Within 120 days of the publication date of this policy, each agency must update—and thereafter keep up to date—its inventory of agency information resources to include an enterprise code inventory that lists custom-developed code for or by the agency after the publication of this policy."
+
+## Configuration
+For documentation on how to configure code-gov-front-end, go [here](CONFIGURATION.md).
+
+## Contributing
+
+Here’s how you can help contribute to code.gov:
+
+* Source Code Policy
+  * To provide feedback on the [Federal Source Code Policy](https://sourcecode.cio.gov/), you should follow [this issue tracker](https://github.com/WhiteHouse/source-code-policy/issues)
+
+* Code.gov
+    * To provide feedback on code-gov-front-end, please checkout our [Contributing Guildelines](CONTRIBUTING.md).
+    * To contribute to the Code.gov style guide, head over to the [code-gov-style](https://github.com/GSA/code-gov-style) repo.
+    * Checkout [code-gov](https://github.com/GSA/code-gov) for a list of additional project repositories. If you aren't sure where your question or idea fits, this is a good place to share it.
+
+## Questions?
+
+If you have questions, please feel free to [send us an email](mailto:code@gsa.gov).
+
+## Getting Started
+
+After you have cloned this repo, you can use `npm install` to install all of the
+project’s dependencies.
+
+You can then run the server using `npm run start`.
+
+By default, the development server will listen on <http://localhost:8080/>. You can change the default port by setting the `PORT` environment variable before starting the server (for example, `PORT=3000 npm start`).
+
+## Testing
```

Comment:
Could you removed this section about tests or say we are currently in the process of writing tests.